### DISCUSSION TITLE
I made a move recommendation AI bot for this game

### DISCUSSION THREAD

**Discussion by Digx7** [ymyke/cardio#124] [General]
Hello ymyke!

I'm a huge fan of Inscryption and love TCG's in general.  For my college AI class I had to make any AI bot I wanted for the final project.  So I decided to make a bot that could do something cool with a TCG.  When I found this repository I initially had hopes of make the bot be able to play the game.  Sadly I had to turn down my scope to meet the deadline.  So instead I made an AI bot that can recommend the best moves for the game.  It was still super fun to work on, and I may improve on it later.  You can find my repository linked below.  Overall, this was a super fun project, Cardio is a really fun game, and I hope you keep working on it.

[AI Final Project Repo](https://github.com/Digx7/AI_Final_Project )

**Comment by ymyke** [comment]
Hey @Digx7, thanks for the heads-up and choosing Cardio for your project! I hope you got a great grade for it! 😉
